packages/trader/src/Modules/SmartChart/index.js

```diff
@@ -18,6 +18,7 @@ export const SmartChart = React.lazy(load('SmartChart'));
 export const AssetInformation = React.lazy(load('AssetInformation'));
 export const ChartTitle = React.lazy(load('ChartTitle'));
 
+export const ChartMode = React.lazy(load('ChartMode'));
 export const ChartSize = React.lazy(load('ChartSize'));
 export const ChartTypes = React.lazy(load('ChartTypes'));
```

Comment:
@shayan-binary-2 does it need to load `ChartTypes` anymore?

Reply:
you are right, we don't need chartType and ChartSize anymore, just I think maybe we use it in another part of the application.